pkg/cli/admin/catalog/mirror.go

```diff
@@ -340,10 +350,10 @@ func (o *MirrorCatalogOptions) Run() error {
 		fmt.Fprintf(o.IOStreams.ErrOut, "errors during mirroring. the full contents of the catalog may not have been mirrored: %s\n", err.Error())
 	}
 
-	return WriteManifests(o.IOStreams.Out, o.SourceRef.Ref.Name, o.ManifestDir, mapping)
+	return WriteManifests(o.IOStreams.Out, o.SourceRef.Ref.Name, o.ManifestDir, o.IcspScope, mapping)
 }
 
-func WriteManifests(out io.Writer, name, dir string, mapping map[string]Target) error {
+func WriteManifests(out io.Writer, name, dir, icspScope string, mapping map[string]Target) error {
```

Comment:
nit: wanna make icspScope an enum?

Reply:
I considered it but doing sting enums doesn't really help much in this case.